contracts/RelaySugar.vy

```diff
@@ -54,61 +64,154 @@ interface IAutoCompounder:
 
 # Vars
 factory: public(IAutoCompounderFactory)
-ve_sugar: public(IVeSugar)
+voter: public(IVoter)
 ve: public(IVotingEscrow)
+token: public(address)
 
 @external
-def __init__(_factory: address, _ve_sugar: address, _ve: address):
+def __init__(_factory: address, _voter: address):
   """
   @dev Set up our external factory contract
   """
   self.factory = IAutoCompounderFactory(_factory)
-  self.ve_sugar = IVeSugar(_ve_sugar)
-  self.ve = IVotingEscrow(_ve)
+  self.voter = IVoter(_voter)
+  self.ve = IVotingEscrow(self.voter.ve())
+  self.token = self.ve.token()
 
 @external
 @view
-def all() -> DynArray[Relay, MAX_COMPOUNDERS]:
+def all(_account: address) -> (DynArray[Relay, MAX_COMPOUNDERS], DynArray[VeNFT, MAX_RESULTS]):
+  """
+  @notice Returns all AutoCompounders and account's deposits
+  @return Array of Relay structs, Array of account's deposits
+  """
+  autocompounders: DynArray[Relay, MAX_COMPOUNDERS] = self._autocompounders()
+  deposits: DynArray[VeNFT, MAX_RESULTS] = self._deposits()
+
+  return autocompounders, deposits
```

Comment:
Since we're returning the `account_venft_ids`, I don't think we need the `deposits`. The idea is that we'll fetch the venfts directly from the `VeSugar` and map it in the dApp with the rest.